core-bundle/src/DependencyInjection/Attribute/AsPage.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\DependencyInjection\Attribute;
+
+/**
+ * An attribute class for page controllers.
+ *
+ * @see \Symfony\Component\Routing\Annotation\Route
+ */
+#[\Attribute(\Attribute::TARGET_CLASS | \Attribute::IS_REPEATABLE)]
+class AsPage
+{
+    public function __construct(public ?string $type = null, public bool $contentComposition = true, public ?string $path = null, public ?string $urlSuffix = null, public array $requirements = [], public array $options = [], public array $defaults = [], public array $methods = [])
+    {
```

Comment:
The `Page` service annotation also has logic in its constructor:

https://github.com/contao/contao/blob/cf615b4640f0a00c1e2a19f5604cb1b86a6d8c4f/core-bundle/src/ServiceAnnotation/Page.php#L39-L57

Is this not required here, too?